Makefile

```diff
@@ -300,27 +294,27 @@ _copy_docs:
 
 # Start a local documentation server.
 serve_docs: _pyspec _copy_docs
-	@$(MKDOCS_VENV) build
-	@$(MKDOCS_VENV) serve
+	@$(UV_RUN) mkdocs build
+	@$(UV_RUN) mkdocs serve
 
 ###############################################################################
 # Checks
 ###############################################################################
 
-MYPY_CONFIG = $(CURDIR)/mypy.ini
-PYLINT_CONFIG = $(CURDIR)/pylint.ini
-
-PYLINT_SCOPE := $(foreach S,$(ALL_EXECUTABLE_SPEC_NAMES), $(PYSPEC_DIR)/eth2spec/$S)
-MYPY_SCOPE := $(foreach S,$(ALL_EXECUTABLE_SPEC_NAMES), -p eth2spec.$S)
 MARKDOWN_FILES := $(shell find $(CURDIR) -name '*.md')
+MYPY_PACKAGE_BASE := $(subst /,.,$(PYSPEC_DIR:$(CURDIR)/%=%))
+MYPY_SCOPE := $(foreach S,$(ALL_EXECUTABLE_SPEC_NAMES), -p $(MYPY_PACKAGE_BASE).eth2spec.$S)
+# Check that lockfile is up to date.
+check-lock:
```

Comment:
This should be made private (`_check-lock`). Preferably, just add it to the lint commands.

Reply:
Removed target, moved command directly to `lint`.